compiler/luci-interpreter/src/kernels/Softmax.test.cpp

```diff
@@ -45,14 +63,41 @@ TEST(SoftmaxTest, Float)
   kernel.configure();
   kernel.execute();
 
-  std::vector<float> ref_output_data{
-      0.38514, 0.09497, 0.51989, //
-      0.20792, 0.51141, 0.28067, //
-      0.25212, 0.18678, 0.56110, //
-      0.48149, 0.19576, 0.32275, //
-  };
-  EXPECT_THAT(extractTensorData<float>(output_tensor),
-              ElementsAreArray(ArrayFloatNear(ref_output_data)));
+  if (std::is_floating_point<T>::value)
```

Comment:
Could you please avoid this construct? It is better to make two tests, one for float and the other for quantized u8.

Reply:
Okay I will fix it!